.bbtravis.yml

```diff
@@ -8,6 +8,7 @@ env:
   matrix:
   - TARGET=qemux86-64
   - TARGET=raspberrypi2
+  - TARGET=intel_nuc-core-i5
```

Comment:
intel-corei7-64 ? as this is the machine name and it should work on any intel core based target ? or I am missing something ?